Changes

```diff
@@ -125,6 +125,9 @@ Working version
   (Jules Aguillon, with help from Armaël Guéneau,
    review by Gabriel Scherer and Florian Angeletti)
 
+- GPR#2309: New option -noruntime
```

Comment:
This could be a little more descriptive! Some mention that stops linking libasmrun for native and suppresses the ocamlrun header for bytecode?

Reply:
Also the manual should be updated.

Reply:
And the man pages in `man/` .